CompositeUi/src/views/component/CardExtended.js

```diff
@@ -13,6 +13,10 @@ import './../../scss/components/_card-extended';
 import React from 'react';
 
 class CardExtended extends React.Component {
+  static propTypes = {
+    thumbnail: React.PropTypes.element
```

Comment:
This is an an antipattern. Here the explanation: https://github.com/facebook/react/pull/7119